src/tox/tox_env/runner.py

```diff
@@ -121,9 +121,10 @@ def _call_pkg_envs(self, method_name: str, *args: Any) -> None:
             with package_env.display_context(suspend=self._has_display_suspended):
                 getattr(package_env, method_name)(*args)
 
-    def _clean(self, force: bool = False) -> None:
-        super()._clean(force)
-        self._call_pkg_envs("_clean")  # do not pass force along, allow package env to ignore if requested
+    def _clean(self, transitive: bool = False) -> None:
+        super()._clean(transitive)
+        if transitive:
+            self._call_pkg_envs("_clean")
```

Comment:
So, do I get this right?  If `_clean` with `transitive` gets called, all existing envs get recreated?

Reply:
Yes all transitive environments. Currently this means packaging environments attached to the run environment. But you might have additional environments attached to the run environment that might need be cleaned.